doc/testing-your-application.rst

```diff
@@ -3,43 +3,26 @@
 Testing Your Application
 ========================
 
-This chapter describes how to test your application against a Varnish instance.
+This chapter describes how to test your application against you reverse proxy
+instance.
 
-VarnishTestCase
----------------
-
-The FOSHttpCache library provides a base test case to help you write functional
-tests against a Varnish instance. This may be helpful to test the way your
-application sets caching headers and invalidates cached content.
+The FOSHttpCache library provides base test case classes to help you write 
+functional tests. This may be helpful to test the way your application sets 
+caching headers and invalidates cached content.
 
-By having your test classes extend ``VarnishTestCase``, you get:
+By having your test classes extend one of the testCase classes, you get:
 
-* independent tests: all previously cached content is removed by restarting
-  Varnish in the test setUp method
-* ``$this->varnish`` referring to an instance of this library’s Varnish client
-  that is configured to talk to your Varnish server
+* independent tests: all previously cached content is removed in the tests 
+  ``setUp`` method. The way this is done depend on which reverse proxy you use.
+* an instance of this library’s client that is configured to talk to your 
+  reverse proxy server. See reverse proxy specific sections for details.
 * convenience methods for executing HTTP requests to your application:
-  ``$this->getHttpClient()`` and ``$this->getResponse()``
+  ``$this->getHttpClient()`` and ``$this->getResponse()``.
```

Comment:
symfony core doc uses `;` for items in bullet points, except the last item which has `.`. i suggest we do the same.